### PR TITLE
Use tiles on the Get Started page

### PR DESCRIPTION
**Pull request by cnunciato**
On the API Reference page, I recently added some "tiles" for the cloud providers. This PR applies that same treatment for the Choose Your Provider selection on the Get Started page, mainly for consistency.

### CODE REVIEW

**Comment by cnunciato** [comment]
@justinvp Oh, awesome -- thanks for this extra context. I'll take a look and see if I can take care of those as well.

File: content/docs/intro/languages/dotnet.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ Pulumi supports infrastructure as code using any .NET Core language. You can use
...
-    <a class="block flex-1 btn bg-transparent border border-solid border-gray-300 hover:bg-gray-200 p-4 mb-4 mr-0 md:mb-0 md:mr-4 flex justify-center" href="{{< relref "../../get-started/azure" >}}">
-        <img class="h-5" src="/logos/tech/azure.svg" alt="Azure">
+    <a class="tile md:mx-4 flex-1 p-6" href="{{< relref "../../get-started/azure" >}}">
+        <img class="h-8 mx-auto" src="/logos/tech/azure.svg" alt="Azure">
     </a>
-    <a class="block flex-1 btn bg-transparent border border-solid border-gray-300 hover:bg-gray-200 p-4 flex justify-center" href="{{< relref "../../get-started/gcp" >}}">
-        <img class="h-5" src="/logos/tech/gcp.svg" alt="Google Cloud">
+    <a class="tile flex-1 p-6" href="{{< relref "../../get-started/gcp" >}}">
+        <img class="h-8 mx-auto" src="/logos/tech/gcp.svg" alt="Google Cloud">
     </a>
```
Comment (justinvp) [inline content/docs/intro/languages/dotnet.md:34]: We've had .NET Kubernetes content in the Getting Started guide, but forgot to go back and add the button here. Mind adding the Kubernetes tile here while making changes here?

Comment (cnunciato) [reply content/docs/intro/languages/dotnet.md]: I totally just did! It's building now.

Comment (justinvp) [reply content/docs/intro/languages/dotnet.md]: Thanks!

**Pull request merged**